hs_tools_resource/templates/pages/toolresource.html

```diff
@@ -42,8 +41,19 @@
                         <div class="row">
                             <div class="col-sm-8">{{ supported_res_types }}</div>
                         </div>
+                        <hr style="border:0">
                     {% endif %}
                 </div>
+                <div>
+                    {% if tool_icon_url %}
+                        <h4><strong>Tool Icon:</strong></h4>
+                        <hr style="margin-top: 0px;margin-bottom: 2px">
+                        <div class="row">
+                            <div class="col-sm-8"><img src="{{ tool_icon_url }}" style="height: 100px; width: 100px;"/></div>
```

Comment:
Same here, you can probably create one css file for your app and include it in hs_core template files as needed to put the styles all together rather than embedding the same style definition in template files in multiple places.